api/resources_portal/models/material.py

```diff
@@ -32,8 +33,14 @@ class Meta:
     mta_s3_url = models.TextField(
         blank=True, null=True, help_text="Contains an url to download the MTA."
     )
+
+    title = models.TextField(
+        blank=False, null=False, help_text="The title of the material.", default="New Material"
```

Comment:
We don't need a default here. If there's no title I'd like to be an error rather than have "New Material" as its title.

Reply:
Removed.

Reply:
Done.